plugins/base/src/test/resources/expect/signatureTest/out/navigation.html

```diff
@@ -0,0 +1,13 @@
+<div class="sideMenuPart" id="nav-submenu">
```

Comment:
Are you sure this is signature specific file? If it does not, it should definitely be moved out of this PR